src/_includes/_layouts/Blog.jsx

```diff
@@ -10,23 +10,141 @@ export default ({ comp, description, posts, children }) => (
     <body>
       <comp.Navbar />
       <main className="max-w-4xl mx-auto p-4">
```

Comment:
Adding onto Christian's comment about the padding on the left and right, here the width is being maxxed out at `max-w-4xl` but that is what's restricting the layout to have such large left and right padding. 

You can change it be something like `max-w-6xl` to reduce the large horizontal space but other things would have to increase in size too to accommodate the white space